src/components/PageDetail.js

```diff
@@ -13,37 +15,63 @@ function PageDetails() {
         
                 }).then(response => {
                     setisData(response.data)
+                    setImages(response.data.user.avatar_url)
+                    setUserLogin(response.data.user.login)
```

Comment:
Can we just remove these line and use the data variable?
Since the data here is redundant.

Reply:
mag error sya sir mao g ani nlang nako sir.

Reply:
![image](https://user-images.githubusercontent.com/109492857/184575809-93110fe4-fd09-4913-a7c7-8027c6576ac9.png)
![image](https://user-images.githubusercontent.com/109492857/184575870-c793e3aa-67cb-4366-a338-ce78d44dcf2a.png)

Reply:
ok na diay ni sir.